native/acter/src/api/room.rs

```diff
@@ -149,12 +151,15 @@ impl Room {
     }
 
     pub async fn get_my_membership(&self) -> Result<Member> {
-        let SdkRoom::Joined(joined) = &self.room else {
+        let room = if let SdkRoom::Joined(r) = &self.room {
```

Comment:
any particular reason you prefer this over the other style?

Reply:
Newer style was introduced by you before.
It looked better to me and I kept that style as much as possible.
That's all.